### ISSUE TITLE
Distinguish "Rebuild" and "not Optimal" for pdisks

### ISSUE THREAD

**Issue by charlesgoyard** [Art3mK/Zabbix-LSI-RAID-Monitoring#6]
Hi Art3mK,

I use your Zabbix module on a lot of servers at work, and the only thing I missed from you excellent work is to be able to separate non-optimal disk from rebuilding disk. This lets my team know the problem has been addressed and we just have to wait.

For raid_trapper__check.pl, this translates into:
```
             my $firmware_state;
             if ($1 =~ /^Rebuild/) {
                 $firmware_state     = 2;
             }
             elsif ($1 =~ m/^(Unconfigured\(good\).*|Online,\sSpun.*|Hotspare,\sSpun.*)$/) {
                 $firmware_state     = 0;
             }
             else {
                 $firmware_state     = 1;
             }

```
And in Zabbix itself I made a "average" trigger for firmware_state=2.

Thanks again !
Charles

**Comment by Art3mK** [comment]
Hey Charles, would you mind to open PR? :) Probably to windows version also?

**Comment by Art3mK** [comment]
fixed by #7, thanks for the PR!
